feature/qos/otg_tests/egress_traffic_classification_and_rewrite_test/README.md

```diff
@@ -0,0 +1,205 @@
+
+* Traffic:
+    * Generate IPv4 traffic from ATE Port 1 with various DSCP values.
+* Verification:
+    * Monitor telemetry on DUT to verify packet scheduling into correct forwarding groups.
+    * Capture packets on ATE's ingress to verify packet re-marking per table.
+    * Analyze traffic flows to confirm no packet drops on DUT.
+
+### DP-1.17.2 Egress Classification and rewrite of IPv6 packets with various TC values
+
+* Traffic:
+    * Generate IPv6 traffic from ATE Port 1 with various TC values
+* Verification:
+    * Monitor telemetry on DUT to verify packet scheduling into correct forwarding groups.
+    * Capture packets on ATE's ingress to verify packet marking per table.
+    * Analyze traffic flows, confirming no drops on DUT.
+
+### DP-1.17.3 Egress Classification and rewrite of IPoMPLSoGUE traffic with pop action
+Member
+@dplore dplore last week
+I think the IPoMPLSoGUE and IPoMPLSoGRE subtests here are possibly duplicates of this test:
+https://github.com/openconfig/featureprofiles/tree/ac98ad2d2a57b460ab6584d4b26f99145a04ba7e/feature/policy_forwarding/otg_tests/mpls_gre_udp_qos
+
+Can you verify that PF-1.18 mpls_gre_udp_qos is a duplicate of those scenarios? IF yes, you can remove the duplicated subtests from this PR.
+
+@axelrod-mike	Reply...
+
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv4/IPv6 forward actions for a specific labels range (100020-100100).
+    *   Configure decapsulation rules for IPv4-DST-DECAP/32
+*   Traffic:
+    *   Generate IPoMPLSoGUE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.4 Egress Classification and rewrite of IPv6oMPLSoGUE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv6 forward actions for a specific labels range (100020-100100).
+    *    Configure decapsulation rules for IPv6-DST-DECAP/12 
+*   Traffic:
+    *   Generate IPv6oMPLSoGUE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.5 Egress Classification and rewrite of IPoMPLSoGRE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv4 forward actions for a specific labels range (100020-100100).
+    *   Configure decapsulation rules for IPv4-DST-DECAP/32
```

Comment:
Again, suggest to mention that we need the rewrite configuration to be specified.